src/main/java/core/basesyntax/JavaDateTimeApi.java

```diff
@@ -74,16 +102,20 @@ public LocalDate addWeeks(LocalDate localDate, Integer numberOfWeeks) {
      * - "someDate is today" - если someDate - сегодня
      */
     public String beforeOrAfter(LocalDate someDate) {
-        return someDate + "is today";
+        LocalDate today = LocalDate.now();
+        return someDate.equals(today) ? someDate + " is today" : someDate.isBefore(today)
+                ? someDate + " is before " + today : someDate + " is after " + today;
     }
 
     /**
      * Дана дата в строковом формате и временная зона.
      * Верните LocalDateTime в этой временной зоне.
+     *
      * @return LocalDateTime
      */
     public LocalDateTime getDateInSpecificTimeZone(String dateInString, String zone) {
-        return LocalDateTime.now();
+        ZonedDateTime zonedDateTime = ZonedDateTime.parse(dateInString + "[" + zone + "]");
+        return LocalDateTime.from(zonedDateTime);
```

Comment:
We can solve it using Instant and ZoneId to avoid   -> "[" + zone + "]"

Reply:
I agree